make_docs.nu

```diff
@@ -1,21 +1,31 @@
-# get all command names from a clean scope
-def command-names [] {
+
+def plugin-paths [ nu_path?: path ] {
     const PLUGINS = [
```

Comment:
These are still hard coded here. I thought we were going to use `plugin list` for these names?

Reply:
Right - We still do have to hardcode the "core" plugin names that we're going to document.  Otherwise we'd also load things like `nu_plugin_stress_internals`.  On the bright side, if any commands are added to the core plugins, they'll get added, documented, and "bannered" automatically.

But yes, if a new "core" plugin is added, removed, or renamed, we'll have to change these hardcoded values.

Your comment does make me think that perhaps `CORE_PLUGINS` would be a better moniker, though.